Requirements for SSE.md

```diff
@@ -37,22 +37,21 @@ TODO: Insert Diagram photo
 
 ### Use Case 2
 
-#### TODO: Insert Use Case Name
+#### Updating Configuration.yaml Use Case
 
-TODO: Insert use-case info
+Bob was a user of a popular application called [IFTTT](https://ifttt.com/) and automated many of his IoT devices until they began their "Pro" subscription service which limited the number of integrations he could use unless he paid IFTTT's monthly subscription fee. Bob recently talked with his neighbor who shared a blog post about [HomeAssistant](https://blog.briancmoses.com/2020/09/ditching-ifttt-for-home-assistant.html) that discussed the advantages of bringing all IoT devices under a single hub. Being a not so savvy tech user Bob began researching, setting up HomeAssistant, and integrating all his IoT devices and hubs. Once Bob has all his IoT devices managed by HomeAssistant he begins creating his own automations similar to what he did on IFTTT. HomeAssistant handles all its automations inside the [Configuration.yaml](https://www.home-assistant.io/docs/configuration/) file which is editable via several official HomeAssistant add-ons: File Editor, Samba, and SSH. Bob enables Samba so he can input all his device information in the Configuration.yaml including his username, API tokens, and password information for his Ring Security System, Blink Camera System, and other IoT device accounts and sets up his automations similar to how he had it in IFTTT. 
```

Comment:
Everything looks good. Reading through this I saw talking should be spoke. Also, no big deal but maybe break up the first sentence. maybe something like this. --> Bob was a user of a popular application called [IFTTT](https://ifttt.com/) and automated many of his IoT devices. They began offering their "Pro" subscription service which limited the number of integrations he could use unless he paid IFTTT's monthly subscription fee.

Reply:
Fixed, nice catch thank you